tidy3d/web/cli/develop/packaging.py

```diff
@@ -0,0 +1,107 @@
+import pathlib
```

Comment:
ss there any mention of these new features in the development guide?

Reply:
Now they're in better